src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -538,11 +538,16 @@ OverlayImpl::onPrepare()
             {
                 if (addr.port () == 0)
                 {
-                    Throw<std::runtime_error> ("Port not specified for "
-                        "address:" + addr.to_string ());
+                    JLOG(journal_.debug()) << "Port not specified (bootstrapIps) for address:" 
```

Comment:
The  comment is no longer relevant. I'd remove it.

Reply:
Removed.